docs/content/practices/emulator_setup.md

```diff
@@ -109,22 +109,28 @@ define_path_environment_if_needed
 create_and_patch_emulator
 ```
 
-Pay your attention that you also need to wait until your emulator is fully booted.
+Keep in mind that the emulator must fully boot before running any test. Otherwise the tests will fail because there is still no device ready
```

Comment:
Actually, for snapshot testing I was using [swarmer](https://github.com/gojuno/swarmer) from Juno, which is deprecated but works pretty fine for starting several emulators and wait for all of them to be ready before running the tests.
Another new thing is Gradle Virtual Managed Devices...

Maybe worth mentioning later, after releasing the MVP?